qcengine/programs/molpro.py

```diff
@@ -45,11 +45,20 @@ class MolproHarness(ProgramHarness):
         "M12HFC", "HJSWPBE", "HJSWPBEH", "TCSWPBE", "PBESOL"
     }
 
-    # Currently supported methods in QCEngine for Molpro
-    _scf_methods: Set[str] = {"HF", "RHF", "KS", "RKS"}
-    _post_hf_methods: Set[str] = {'MP2', 'CCSD', 'CCSD(T)'}
-    _supported_methods: Set[str] = {*_scf_methods, *_post_hf_methods}
-
+    # Different open-shell scenarios:
+    # - Restricted references
+    #       RHF-RMP2 (like RHF-UMP2 using Molpro's naming convention for CC methods)
+    #       RHF-UCCSD
+    #       RHF-UCCSD(T)
+    #       RHF-RCCSD
+    #       RHF-RCCSD(T)
```

Comment:
btw, if you want to join in the "standard_suite" make-the-programs-match fun, https://lists.uni-mainz.de/sympa/arc/cfour/2019-09/msg00030.html may be handy for us all.